tests/test_sqlalchemy_data_layer.py

```diff
@@ -172,7 +172,7 @@ def test_get_list(client, registered_routes, person, person_2):
             {
                 "page[number]": 1,
                 "page[size]": 1,
-                "fields[person]": "name,birth_date",
+                "fields[person]": "name,birth_date,computers",
```

Comment:
I think because `computers` is a related resource, not a field, this won't do anything. Does it?

Reply:
But `computers` will treat like a field in `marshmallow-jsonapi `

Reply:
If not add `computers` to the fields it will throw error `Unknown field`, because `only` will exclude the `computers` in `marshmallow-jsonapi `. I feel confuse how to handle it, the request wants `only` name and birth but include `computers.owner`, should it return computers data ? 😂

Reply:
I find out some details in [docs](https://flask-rest-jsonapi.readthedocs.io/en/latest/sparse_fieldsets.htm), that fields should include `computers `.

> Warning: If you want to use both “fields” and “include” don’t forget to specify the name of the relationship in fields; if you don’t the include wont work.

Reply:
Good find! And that is supported by the spec, which says relationships are a type of `field: https://jsonapi.org/format/#fetching-sparse-fieldsets.